package.json

```diff
@@ -6,15 +6,16 @@
   "bugs": {
     "url": "https://github.com/FormidableLabs/next-urql/issues"
   },
-  "main": "dist/next-urql.min.js",
-  "module": "dist/next-urql.es.min.js",
+  "main": "dist/next-urql.js",
+  "module": "dist/next-urql.es.js",
   "source": "src/index.ts",
-  "types": "index.d.ts",
+  "types": "dist/types/index.d.ts",
   "author": "Formidable",
   "license": "MIT",
   "scripts": {
     "build": "rollup -c rollup.config.js",
-    "check": "run-s check:types test lint format",
+    "check:ci": "run-s check:types lint format test",
```

Comment:
Is the purpose of this to run the steps that CI runs?

Reply:
Yep, but it's more a local check for our benefit so you don't manually have to type them out. In CI it's better practice to specify each step separately so that it's easier to identify which one failed. If we ran just this script in CI, it could be any one of these steps that failed; breaking them out makes them easier to identify.